website/src/static/xviz-docs

```diff
@@ -0,0 +1 @@
+../../../../xviz/docs
```

Comment:
These unfortunately will be copied when we build the website. Can you move them out of the `static/` directory?